apps/web/e2e/auth.setup.ts

```diff
@@ -0,0 +1,49 @@
+import { test as setup, expect } from "@playwright/test";
+import { signInWithEmail } from "./utils/auth-test";
+import { generateTestUserData } from "./utils/test-data";
+
+const authFile = "playwright/.auth/user.json";
+
```

Comment:
remove comments and just create ONE comments at the ebgginer that explain in general what appends